Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -704,8 +727,10 @@ public final class ManifestLoader: ManifestLoaderProtocol {
         packageIdentity: PackageIdentity,
         toolsVersion: ToolsVersion,
         delegateQueue: DispatchQueue,
-        result: inout EvaluationResult
+        completion: @escaping (Result<EvaluationResult, Error>) -> Void
```

Comment:
similar to https://github.com/apple/swift-package-manager/pull/3872/files#r750487258

Reply:
Fixed in fd4f7e27.